### DISCUSSION TITLE
Another Vercel account is using this domain.另一个Vercel帐户正在使用此域。

### DISCUSSION THREAD

**Discussion by AM-QAQ** [vercel/community#7101] [Help] [answered]
### Summary

Verification has been unsuccessful while adding domain names to projects on Vercel.
在为Vercel上的项目添加域名时，验证一直无法通过。

### Steps to Reproduce

![1](https://github.com/vercel/community/assets/147171917/5457c257-69c2-417f-ad0a-c3084f0acd2e)
![2](https://github.com/vercel/community/assets/147171917/64cbf9ef-5e92-4796-a286-20accee55e6c)

After adding TXT, it still shows "domain name waiting for verification". What is the problem.
在添加TXT后仍然显示“域名正在等待验证”，请问这是什么问题。

**Comment by amyegan** [comment, marked as answer]
There is currently no TXT record on `_vercel.cloudns.ch` (https://www.whatsmydns.net/#TXT/_vercel.cloudns.ch).

Root domain access must be verified before the `gemini` subdomain can be used on your project. Make sure you have set the correct name and value for the TXT record.

**Reply by AM-QAQ** [reply]
thanks
